september-util/src/main/java/edu/asu/diging/gilesecosystem/septemberutil/service/impl/SystemMessageHandler.java

```diff
@@ -50,6 +50,15 @@ public SystemMessageHandler(String applicationId) {
 
     @Override
     public void handleError(String msg, Exception exception) {
+        handleCommonException(msg, exception, ISystemMessageRequest.ERROR);
+    }
+
+    @Override
+    public void handleWarning(String msg, Exception exception) {
+        handleCommonException(msg, exception, ISystemMessageRequest.WARNING);
+    }
+
+    private void handleCommonException(String msg, Exception exception, String messageType) {
```

Comment:
why don't you just pass the message type as parameter in the handle method? This way you never need to touch it again, no matter how many different message types there are or will be.